src/calculateRentalCost.js

```diff
@@ -4,7 +4,33 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  // I tried everyting, and there is actualy second way to solve this problem
+  // Well, this program gets job done.
+
+  const LONG_TERM = 7;
+  const MED_TERM = 6;
+  const SHORT_TERM = 3;
+
+  const BASE_PRICE = 80;
+  const SHORT_TERM_DISCOUNT = -20;
+  const MID_TERM_DISCOUNT = -140;
+  const LONG_TERM_DISCOUNT = -150;
```

Comment:
The constants for term definitions are correct, but the BASE_PRICE and discount values are not aligned with the task description. The BASE_PRICE should be $40 per day, and the SHORT_TERM_DISCOUNT should be $20, not negative. The MID_TERM_DISCOUNT and LONG_TERM_DISCOUNT should not be negative and must correspond to the task description of $20 and $50 off, respectively.